docs/data/joy/components/aspect-ratio/CarouselRatio.tsx

```diff
@@ -52,7 +52,11 @@ export default function FlexRowRatio() {
             ratio="1"
             sx={{ minWidth: 60, borderRadius: 'sm', overflow: 'auto' }}
           >
-            <img src={item.src} alt={item.title} />
+            <img
+              src={`${item.src}?h=120&fit=crop&auto=format`}
+              srcSet={`${item.src}?h=120&fit=crop&auto=format&dpr=2 2x`}
```

Comment:
See the demos in https://mui.com/material-ui/react-masonry/#image-masonry for the gold standard.